invenio_rdm_migrator/state.py

```diff
@@ -193,11 +193,20 @@ def _initialize_db(cls, metadata):
             sa.Column("id", UUIDType, nullable=False, unique=True),
             sa.Column("parent_id", UUIDType, nullable=False),
             sa.Column("index", sa.Integer, nullable=False),
+            # record's version_id
             sa.Column("fork_version_id", sa.Integer, nullable=False),
             sa.Column("pids", JSONType, default="{}"),
             sqlite_autoincrement=False,
         )
 
+        sa.Table(
```

Comment:
In the file actions the lookup is done by bucket id, therefore it makes more sense to store the bucket-draft mapping rather than draft information (we would need then the inverted mapping or custom logic on this state entity).

Reply:
Ok, I think I'm cool with having specialized state tables. I know originally we wanted to kind of have individual entities that map 1-1 with what's on RDM, but let's see how it evolves.